docker/checkout.bash

```diff
@@ -40,11 +40,11 @@ cd ${dir}/src
 if [[ "$BRANCH" = "develop" ]]; then
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch develop
       git clone --depth=1 https://github.com/usdot-fhwa-stol/novatel_gps_driver.git --branch develop
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch develop
+      git clone --depth=1 https://github.com/mjeronimo/carma-utils.git --branch feature/ros-noetic-port
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-messenger.git --branch develop
 else
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch develop
       git clone --depth=1 https://github.com/usdot-fhwa-stol/novatel_gps_driver.git --branch develop
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch develop
+      git clone --depth=1 https://github.com/mjeronimo/carma-utils.git --branch feature/ros-noetic-port
```

Comment:
Need to remember to update these links once the respective repos are merged

Reply:
@msmcconnell I'll update. I suppose these should all be "noetic/develop", correct? Same goes for all other checkout.bash files in other repos.

Reply:
correct